application/src/main/java/org/opentripplanner/transit/model/basic/Distance.java

```diff
@@ -23,8 +24,22 @@ public static Distance ofKilometers(double value) {
   }
 
   /** Returns the distance in meters */
-  public double toMeters() {
-    return this.meters;
+  public int toMeters() {
+    return (int) Math.round(this.meters);
+  }
+
+  /**
+   * Is this distance greater than the one passed in
+   */
+  public boolean greaterThan(Distance distance) {
+    return this.meters > distance.meters;
+  }
+
+  /**
+   * Is this distance less than the one passed in
+   */
+  public boolean lessThan(Distance distance) {
+    return this.meters < distance.meters;
```

Comment:
Is there any other use for these than tests? However, I can see how these could be useful in the future for something.

Reply:
No other uses for now, I added them to test the conversion meter - km (even though equals is sufficient) and in the event that they will be useful in the future. 
I can easily get rid of those

Reply:
Yeah, we probably don't want to add unused code.